### PR TITLE
Add newsletter component (#48)

### PR DESCRIPTION
**Pull request by pranav2012**
## What type of PR is this? (check all applicable)
- [x] ✨ Feature
- [x] 👷 Optimization

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
image

## Mobile

image

### CODE REVIEW

**Review by bobbyiliev — CHANGES_REQUESTED** [review]
Overall this looks good, but note that the component needs to be written in [TailwindCSS](http://tailwindcss.com/), so we can't merge it as it is unless you rework it in TailwindCSS.

Also no need to include the JS part, only the TawindCSS component is enough.

Thanks 🙌

**Comment by connecteev** [comment]
@pranav2012 Also any chance you could attach some screenshots when this is redone in Tailwindcss only?

**Comment by pranav2012** [comment]
added

**Comment by pranav2012** [comment]
Hey, @bobbyiliev @thinkverse Please review and merge added screenshots and Tailwind CSS and also removed the js.Thank you:smile:

File: components/newsletter.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+                <p class= "text-sm ml-1 text-indigo-200">SinUp for our newsletter to stay up to date.</p>
+            </div>
+            <form method="POST">
+                <div class="flex p-4">
+                    <input id="i" class="border-none text-sm rounded w-3/4 text-center mr-2" type="email" placeholder="your email address">
+                    <button class="bg-blue-400 font-extrabold hover:bg-blue-500 text-white text-sm w-32 h-8 rounded-md" type="submit">Notify Me</button>
+                </div>
+            </form>
+        </div>
+    </div>
```
Comment (bobbyiliev) [inline components/newsletter.html]: Looks good to me, just one change request on my side. Can you remove the body and the head tags? Only this div is enough 🙌

Comment (pranav2012) [reply components/newsletter.html]: sure :+1:

**Comment by pranav2012** [comment]
@bobbyiliev done removed the tags

**Review by tnylea — CHANGES_REQUESTED** [review]
Very cool, if you could make this responsive, I can merge it in and then make a few touch-ups :)

Thanks!

Screen Shot 2020-10-01 at 9 58 27 AM

Screen Shot 2020-10-01 at 9 58 39 AM

**Comment by pranav2012** [comment]
@tnylea , made it responsive :smile:

**Comment by pranav2012** [comment]
@thinkverse  @tnylea @bobbyiliev please Review and merge my pull request updated with all the requirement's mentioned, Thank you :blush:

**Review by bobbyiliev — APPROVED** [review]
Looks good

**Comment by pranav2012** [comment]
Thanks @bobbyiliev and everyone 😊

**Pull request merged**